### PR TITLE
Avinash setup tdd

### PR DESCRIPTION
**Pull request by techoavinash**
@paweltechie , @techolution-ex 

this branch is for setup and League Service TDD

please review

### CODE REVIEW

File: league/src/test/java/com/makeurpicks/service/LeagueServiceTest.java [hunk 1]
```diff
@@ -1,16 +1,222 @@
...
+		leagueService.createLeague(league);
+		verify(leagueRepositoryMock).save(league);
+	}
+
+	@Test
+	public void createLeague_addPlayerToLeague() {
+		League league = getLeagueFull();
+		when(leagueRepositoryMock.findOne(UUID.randomUUID().toString())).thenReturn(league);
+		leagueService.createLeague(league);
+	}
```
Comment (paweltechie) [inline league/src/test/java/com/makeurpicks/service/LeagueServiceTest.java]: There is not test in this method

File: league/src/test/java/com/makeurpicks/service/LeagueServiceTest.java [hunk 2]
```diff
@@ -1,16 +1,222 @@
...
+				.withSeasonId(seasonId).withNoSpreads().build();
+		when(leagueRepositoryMock.findOne(leagueId)).thenReturn(league);
+		leagueService.updateLeague(league);
+		verify(leagueRepositoryMock).save(league);
+	}
+
+	// Method - updateLeague(League league) - Negative test cases
+	@Test
+	public void updateLeague_leagueDoesnotExist() {
+		expectedEx.expect(LeagueValidationException.class);
```
Comment (paweltechie) [inline league/src/test/java/com/makeurpicks/service/LeagueServiceTest.java:178]: You should look here also for the specific exception message.

File: league/src/test/java/com/makeurpicks/service/LeagueServiceTest.java [hunk 3]
```diff
@@ -1,16 +1,222 @@
...
+		when(playerLeagueRepository.findByIdLeagueIdAndIdPlayerId(league.getId(), playerId)).thenReturn(playerLeague);
+		leagueService.removePlayerFromLeague(leagueId, playerId);
+		verify(playerLeagueRepository).delete(playerLeague);
+	}
+
+	// Method - removePlayerFromLeague(String leagueId, String playerId) -
+	// Negative test cases
+	@Test
+	public void removePlayerFromLeague_invalidLeague() {
+		expectedEx.expect(LeagueValidationException.class);
```
Comment (paweltechie) [inline league/src/test/java/com/makeurpicks/service/LeagueServiceTest.java:221]: Here also, check for specific exception message

File: league/src/test/java/com/makeurpicks/service/LeagueServiceTest.java (hunk 1 above)
Comment (techoavinash) [reply league/src/test/java/com/makeurpicks/service/LeagueServiceTest.java]: updated this method to be a test method

File: league/src/test/java/com/makeurpicks/service/LeagueServiceTest.java (hunk 2 above)
Comment (techoavinash) [reply league/src/test/java/com/makeurpicks/service/LeagueServiceTest.java:178]: incorporated message also to be verified

File: league/src/test/java/com/makeurpicks/service/LeagueServiceTest.java (hunk 3 above)
Comment (techoavinash) [reply league/src/test/java/com/makeurpicks/service/LeagueServiceTest.java:221]: incorporated message to verify

**Pull request closed without merging**